kong/templates/nginx_kong.lua

```diff
@@ -121,6 +121,7 @@ server {
 
         proxy_http_version 1.1;
         proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-Host $host;
```

Comment:
On the most recent master, there is already a header called `X-Forwarded-Host`: https://github.com/Mashape/kong/blob/master/kong/templates/nginx_kong.lua#L126